src/tools/autocompletion/README.md

```diff
@@ -0,0 +1,15 @@
+# Autocompletion
+
+To get autocompletion to run you need to have python3 installed and the kdb-python Module up and running.
+For the kdb-python Module to run, the `~/.bashrc` needs to contain `export PYTHONPATH=${PYTHONPATH}:/PATH/TO/PYTHON`.
+
+## Bash
+
+## How to get it to run
+First you need to mount the specification file with kdb by running `sudo kdb mount ~/PATH/TO/SPEC/FILE/spec_ spec/tests/autocomplete -f ni`.
```

Comment:
```suggestion
First you need to mount the specification file with kdb by running `sudo kdb mount /ABSOLUTE/PATH/TO/SPEC/FILE spec/tests/autocomplete -f ni`.
```

Reply:
I would prefer if you say that you assume that someone is in the folder and then use `$(pwd)/spec` to refer to the spec file.

Reply:
(Of course you need to explicitly write down that we assume someone is in this folder.)